dashboard/config/scripts/unique_level_name_here.multi

```diff
@@ -7,4 +7,4 @@ right 'right answer'
 
 markdown <<MARKDOWN
 
-MARKDOWN
```

Comment:
Having a hard time getting rid of these changes, I can make sure they don't cause trouble on their way out to LB